app/models/piece.rb

```diff
@@ -67,4 +68,9 @@ def path_obstructed?(x, y)
     return diagonal_obstructed?(x, y) if diagonal_move?(x, y)
     false
   end
+
+  def opposing_color
```

Comment:
Oops! Nothing yet. I planned to have it return the opposite of this piece's color but didn't finish it and got distracted!